configure.ac

```diff
@@ -1048,6 +1048,36 @@ AM_CONDITIONAL([OPENTHREAD_ENABLE_RAW_LINK_API], [test "${enable_raw_link_api}"
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_RAW_LINK_API],[${OPENTHREAD_ENABLE_RAW_LINK_API}],[Define to 1 if you want to enable raw link-layer API])
 
 #
+# Passing up ACK frame (This option should be removed after all platforms could pass up ACK frame)
+#
+
+AC_ARG_ENABLE(passing_up_ack,
+    [AS_HELP_STRING([--enable-passing-up-ack],[Enable passing up ack support @<:@default=yes@:>@.])],
```

Comment:
Since this option is temporary, how about simply including `#define` in the project-specific core configuration header files, such as [this](https://github.com/openthread/openthread/blob/master/examples/platforms/nrf52840/openthread-core-nrf52840-config.h)?

We could have it where only the "legacy" code needs to define something like `OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE=1`.  If that is not defined, then we default to the new code here.